js/views/PageNav.js

```diff
@@ -138,7 +137,10 @@ export default class PageNav extends View {
     });
 
     this.$addressBar = this.$('.js-addressBar');
+    this.$navList = this.$('.js-navList');
+    this.$popMenus = this.$('.js-navPopMenu');
 
+    getBody().on('click', (e) => { this.onBodyClick(e); });
```

Comment:
Why the extra wrapping function? Why not just `getBody().on('click', this.onBodyClick);`?

This shouldn't happen in render, otherwise if we re-render it's going to be bound again and we'll have duplicate handlers. If you do want it to be in render, then at least we should ensure we're removing any existing handlers before adding the new one (`.off(this.onClick)`).

Also, kind of minor, but I would just do this on document and not body. That's just usually where I've seen global handlers added, but if you really prefer body, no problem.

Also, this event handler would need to be removed in remove(). (I know that we're probably never going to remove this view, but IMHO, I still think it's good practice to have a view clean up after itself. Then, if for some reason we ever do have to remove it, we don't have to try and figure out all the cleanup things.)